modules/npm/src/main/typescript/main.d.ts

```diff
@@ -0,0 +1,3 @@
+declare class Foo {
```

Comment:
Oh, this is missing an `export`. Oh well :)
```suggestion
export declare class Foo {
```